tests/test_base.py

```diff
@@ -85,7 +248,7 @@ def test_gaussian():
             myImg.array, savedImg.array, 5,
             err_msg="Using GSObject Gaussian disagrees with expected result")
     np.testing.assert_almost_equal(
-            myImg.array.sum(dtype=float) *dx**2, myImg.added_flux, 5,
+            myImg.array.sum() *dx**2, myImg.added_flux, 5,
```

Comment:
I don't think you meant to change this.  It looks like you had some kind of merge error and ended up committing back to an earlier version of a file from here down.  You should copy back in the master version of this file for the parts that aren't new for DeltaFunction.